components/alert.scss

```diff
@@ -1,6 +1,9 @@
 $fd-icons-path: "../scss/icons/";
 $fd-fonts-path: "../scss/fonts/";
 
+@import "./../node_modules/@sap-theming/theming-base-content/content/Base/baseLib/sap_fiori_3/sass_variables.scss";
+@import "./../node_modules/@sap-theming/theming-base-content/content/Base/baseLib/sap_fiori_3/css_variables";
+
```

Comment:
Do we need the `scss` reference here if it's already defined in settings that are being imported into alert.scss?

Reply:
We don't need it, I removed it. Thanks for noticing it

Reply:
We don't need it, I removed it. Thanks for noticing it